js/index.js

```diff
@@ -41,6 +41,30 @@ $("#submit").click(function(e) {
     }
 });
 
+$("#fb-login").click(function(e) {
+    e.preventDefault();
+    if(isFacebookSignedIn){
+        $('html, body').animate({
+            scrollTop: $("#form").offset().top
+        }, 2000);
+
+    }
+    else{
+        FB.login(function(response){
+            statusChangeCallback(response);
+            if (response.status == 'connected') {
+                $('html, body').animate({
+                    scrollTop: $("#form").offset().top
+                }, 2000);
+            } else {
+                //TODO: should prompt any error message?
```

Comment:
Won't. The major error here due to the user disagrees the login. Just little exception, so I think we can neglect it